config-schema.json

```diff
@@ -0,0 +1,344 @@
+{
+	"type": "object",
+	"properties": {
+		"server": {
+			"type": "object",
+			"properties": {
+				"host": {
+					"type": "string"
+				},
+				"port": {
+					"type": "integer"
+				},
+				"apiKey": {
+					"type": "string",
+					"minLength": 16,
+					"maxLength": 16
+				},
+				"basicAuth": {
+					"type": "object",
+					"properties": {
+						"user": {
+							"type": "string"
+						},
+						"pass": {
+							"type": "string"
+						}
+					},
+					"required": [
+						"user",
+						"pass"
+					]
+				}
+			},
+			"required": [
+				"host",
+				"port"
+			],
+			"oneOf": [
+				{
+					"required": [
+						"apiKey"
+					]
+				},
+				{
+					"required": [
+						"basicAuth"
+					]
+				}
+			]
+		},
+		"schedule": {
+			"type": "string",
+			"pattern": "(@(annually|yearly|monthly|weekly|daily|hourly|reboot))|(@every (\\d+(ns|us|µs|ms|s|m|h))+)|((((\\d+,)+\\d+|(\\d+(/|-)\\d+)|\\d+|\\*) ?){5,7})"

```

Comment:
This currently fails for me when schedule is `schedule: "0 */6 * * *"`. So we either have to adjust the regex, or update `config.yaml` to say `@every 6h`, or remove the whitespaces. I like the `@every 6h` approach, easier to understand.

So the example in [README.md](https://github.com/autobrr/omegabrr?tab=readme-ov-file#config) should be updated as well.

Reply:
Ah ok, that's an oversight on my part. I'll modify it so it works with both, but I do agree that the `@every 6h` approach is easier to understand.

Reply:
Updated regex. It should now work in both cases